### PR TITLE
Added basic management command for managing org tasks

### PR DESCRIPTION
**Pull request by rowanseymour**
Maybe at some point it's worth building this into a web view of the tasks, but for now this makes it easy to enable and disable org specific tasks.

### CODE REVIEW

File: dash/orgs/management/commands/orgtasks.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        parser.add_argument('action', choices=self.ACTION_CHOICES, help="The action to perform")
+        parser.add_argument('task', nargs='?', default=None, help="Key of a task")
+        parser.add_argument('org_ids', metavar='ORG', type=int, nargs='*', help="The ids of the orgs")
+
+    def handle(self, *args, **options):
+        org_ids = options['org_ids']
+        action = options['action']
+        task_key = options['task']
+
+        orgs = Org.objects.filter(is_active=True).order_by('pk')
```
Comment (norkans7) [inline dash/orgs/management/commands/orgtasks.py]: may be remove is_active filter, I think this management should be possible for all orgs

File: dash/orgs/management/commands/orgtasks.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    def handle(self, *args, **options):
+        org_ids = options['org_ids']
+        action = options['action']
+        task_key = options['task']
+
+        orgs = Org.objects.order_by('pk')
+        if org_ids:
+            orgs = orgs.filter(pk__in=org_ids)
+        else:
+            orgs = orgs.filter(is_active=True)
```
Comment (norkans7) [inline dash/orgs/management/commands/orgtasks.py]: same idea, I think considering all orgs even inactive one

Comment (rowanseymour) [inline dash/orgs/management/commands/orgtasks.py]: I think for the listing commands it doesn't make sense to include inactive orgs by default, and for the enable/disable commands, it's likely that old inactive orgs won't even have those tasks

Comment (norkans7) [inline dash/orgs/management/commands/orgtasks.py]: Okay

**Pull request merged**